### PR TITLE
Add metric for known clients.

### PR DESCRIPTION
**Pull request by SuperQ**
Signed-off-by: Ben Kochie <superq@gmail.com>

### CODE REVIEW

**Comment by SuperQ** [comment]
@brian-brazil Slightly hacky, but it seemed the least complicated way to do this. I was thinking about doing this via Const, but it felt like a lot of overhead to make it that way.

File: proxy/coordinator.go [hunk 1]
```diff
@@ -13,13 +13,29 @@ import (
...
 var (
 	registrationTimeout = kingpin.Flag("registration.timeout", "After how long a registration expires.").Default("5m").Duration()
 )
 
+// Coordinator metrics.
+var (
+	knownClients = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Namespace: namespace,
+			Name: "known_clients",
```
Comment (brian-brazil) [inline proxy/coordinator.go]: Just "clients" would do I think. You can also use promauto to save the manual register.

Comment (SuperQ) [reply proxy/coordinator.go]: Fixed.

**Comment by brian-brazil** [comment]
Tests are failing, looks like a vendoring issue.

**Comment by SuperQ** [comment]
Fixed, also bumped the client_golang up to 1.0.0.

**Pull request merged**